packages/models/src/contract/index.ts

```diff
@@ -1 +1,95 @@
 // TODO: https://github.com/ckb-js/kuai/issues/3
+import { ActorURI, MessagePayload } from "../actor";
+import Store from "../store/store";
+
+enum MethodResponseStatus {
+  Failed = 0,
+  Success
+}
+
+interface ContractMethodResponse {
+  status: MethodResponseStatus
+  reason?: any
+  data?: any
+}
+
+type MethodType = 'call' | 'cast'
+
+interface ContractMethodInterface {
+  [key: ActorURI]: {
+    payload: MessagePayload
+    result?: ContractMethodResponse
+  }
+}
+
+export interface State {}
+
+export default class Contract extends Store {
+
+  update(oldState: State): State {
```

Comment:
Would it be better to type `State` as generic

Reply:
BTW, this PR is specific for `Store` model. Why is the `contract` model included here